refactor: align color scheme and container (#3770)
db-ui/mono: docs/migration/v0.7.x-to-v1.0.0.md

## Migration Beta (0.7.x) ➡ 1.0.0

### Foundations

We update some classes and `data-`-attributes to match with the "Appearances" inside Figma:

- `data-color-scheme` ➡️ `data-mode` - if you set a container to a fixed `dark` or `light` mode
- `data-container-color` ➡️ `data-color` - if you changed an adaptive color scheme
- ❌ `data-color="XXX-bg-basic-level-[1|2|3]"` - removed combination of background and adaptive color scheme

## Automate migration via CLI

We provide a CLI tool to auto migrate your source code. Use this command in your repository:

```shell
npx @db-ui/migration --type=v007_v100 --src=./src
```

> Please check the changes made in your codebase afterwards, as this is mainly a simple search & replace and there might be unexpected changes of similar wordings to our icon names for any other methods, or further code occurrences that don't even refer to icons.

Reviewer: ```suggestion
Please check the changes made in your codebase afterwards, as this is mainly a simple search & replace and there might be unexpected changes of similar wordings to our color names for any other methods, or further code occurrences that don't even refer to colors.
```

> Please check the changes made in your codebase afterwards, as this is mainly a simple search & replace and there might be unexpected changes of similar wordings to our color names for any other methods, or further code occurrences that don't even refer to colors.

Reviewer: ```suggestion
Please check the changes made in your codebase afterwards, as this is mainly a simple search &amp; replace and there might be unexpected changes of similar wordings to our color names for any other methods, or further code occurrences that don't even refer to colors.
```

> Please check the changes made in your codebase afterwards, as this is mainly a simple search &amp; replace and there might be unexpected changes of similar wordings to our color names for any other methods, or further code occurrences that don't even refer to colors.

Reviewer: ```suggestion
Please check the changes made in your codebase afterwards, as this is mainly a simple search & replace and there might be unexpected changes of similar wordings to our color names for any other methods, or further code occurrences that don't even refer to colors.
```
